app/src/main/res/layout/onboarding_fragment.xml

```diff
@@ -0,0 +1,142 @@
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/oppiaBackgroundYellowIvory">
+
+    <androidx.viewpager.widget.ViewPager
+      android:id="@+id/onboarding_slide_view_pager"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:layout_marginBottom="36dp"
+      android:overScrollMode="never"
+      android:scrollbars="none"
+      app:layout_constraintBottom_toTopOf="@id/bottom_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/get_started_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="56dp"
+      android:layout_marginEnd="56dp"
+      android:layout_marginBottom="36dp"
+      android:visibility="@{viewModel.slideNumber == 3 ? View.VISIBLE: View.GONE, default=gone}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent">
+
+      <Button
+        android:id="@+id/get_started_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/colorPrimary"
+        android:fontFamily="sans-serif-medium"
+        android:minHeight="48dp"
+        android:onClick="@{(v) -> presenter.clickOnGetStarted()}"
+        android:text="@string/get_started"
+        android:textColor="@color/white"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/bottom_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="36dp"
+      android:layout_marginEnd="8dp"
+      android:visibility="@{viewModel.slideNumber != 3 ? View.VISIBLE: View.GONE, default=visible}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent">
+
+      <LinearLayout
+        android:id="@+id/slide_dots_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <View
```

Comment:
Can this be made dynamic. If suppose onboarding screens are increased or decreased depending on any of the new feature, then we will have to add these views accordingly. Just a suggestion.

Reply:
@veena14cs Nice point. I will update the implementation accordingly.

Reply:
Done. Thanks.